python/lsst/ts/scheduler/driver/feature_scheduler.py

```diff
@@ -300,6 +300,41 @@ def select_next_target(self):
 
         return self._handle_desired_observation(desired_observation=desired_obs)
 
+    def select_next_targets(self) -> list[FeatureSchedulerTarget]:
+        """Picks a target and returns it as a target object.
```

Comment:
Please use an active voice, i.e. `Pick a target and return it as a target object.`.